lib/Dialect/SV/Transforms/PrettifyVerilog.cpp

```diff
@@ -243,9 +244,51 @@ void PrettifyVerilogPass::sinkExpression(Operation *op) {
   anythingChanged = true;
 }
 
+/// Replace operands which are driven by named wires with inner symbols with
+/// reads of the wires.  This is a heurstic that results in good "named" wires
+/// being used and acts to ensure that preserved Chisel names are actually used
+/// where possible.
+void PrettifyVerilogPass::useNamedOperands(
+    Operation *op, DenseMap<Value, Operation *> &pipeMap) {
+  auto wire = dyn_cast<sv::WireOp>(op);
+  if (!wire || !wire->hasAttr("inner_sym"))
+    return;
+
+  auto users = wire->getUsers();
+  if (std::distance(users.begin(), users.end()) != 1)
```

Comment:
users.begin() + 1 == users.end().  Distance is O(n) for linked lists.

Reply:
Ack, good catch!

Reply:
Fixed in the force push.